app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Removed unused tools

Reply:
I urge caution making changes to the AndroidManifest.xml.  This file describes the program.  Most items are here for a reason.  Are you certain they are not used?  Is it a best practice to remove them?